core/src/main/scala/kafka/tools/StorageTool.scala

```diff
@@ -56,6 +55,11 @@ object StorageTool extends Logging {
       formatParser.addArgument("--ignore-formatted", "-g").
         action(storeTrue())
 
+      if (args.length == 0 || args.contains("--help")) {
```

Comment:
this should be first line inside try block?

Reply:
@seanli-rallyhealth Maybe let it here is ok, since the above code is just create parser, it will not throw exception, and below code will begin to parse arguments.